moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -388,6 +388,12 @@ export const useColumns = ({ hiddenColumns, linkStateFilters, classes }) =>
         hidden: hiddenColumns["interim_project_id"],
         emptyValue: "-",
       },
+      {
+        title: "Components",
+        field: "components",
+        hidden: hiddenColumns["components"],
+        emptyValue: "-",
+      },
```

Comment:
would you make these render with line breaks (like project team members, etc)?

Reply:
added a filter to split the string into an array, and display as block